test/cmd/builds.sh

```diff
@@ -12,16 +12,64 @@ os::log::install_errexit
 url=":${API_PORT:-8443}"
 project="$(oc project -q)"
 
+# expect_build_output_spec runs `oc new-build` with given arguments and expects
+# a build config with a given name to be created with a given build output spec
+function os::cmd::builds::expect_build_output_spec() {
```

Comment:
I think an `os::cmd::expect_success` on the `new-build`, a function to build the correct template, and a `os::cmd::expect_success_and_text` on the `get` with the template should suffice - I'm not sure making the tests less explicit is worth it.